docs/src/assets/scss/components/docs-index.scss

```diff
@@ -152,3 +157,33 @@
         }
     }
 }
+
+.enhanced .switcher-fallback {
+    display: none !important 
```

Comment:
I’m not a fan of needing to rely on !important everywhere.

Could we simplify all of this logic by maybe creating .js-only and .js-only-medium classes that we can just apply in the HTML?